app/views/state_file/questions/w2/edit.html.erb

```diff
@@ -9,17 +9,19 @@
       <p class="spacing-below-25"><%= @w2.employer_name %></p>
 
       <% if @box14_codes.present? %>
-        <p class="spacing-below-5"><%= t(".box14_html") %></p>
-        <% @box14_codes.each do |code| %>
-          <div class="form-question spacing-below-25">
-            <% field_name = "box14_#{code.downcase}" %>
-            <%= f.vita_min_money_field(
-                  field_name.to_sym,
-                  t(".box14_#{code.downcase}_html"),
-                  classes: ["form-width--long"]
-                ) %>
-              <%= t(".box14_#{code.downcase}_help_text", year: MultiTenantService.statefile.current_tax_year) %>
-          </div>
+        <fieldset>
```

Comment:
Grouping related fields semantically

Reply:
could you add a screenshot to the PR description of what this looks like now? (and if you have time, maybe also one for MD because it's the only other state with a box 14 code) i assume it looks similar but since there are changes to the view, would want to see that it at least didn't change

Reply:
Added before and after screenshots for MD. There weird scroll thing happening that only shows up in the screenshot, but box 14 stuff looks the same